main/src/modules/abm/abmReasoning/src/abmReasoning.cpp

```diff
@@ -186,6 +186,8 @@ bool abmReasoning::respond(const yarp::os::Bottle& bCommand, yarp::os::Bottle& b
 {
 
     bReply.clear();
+    bReply.addVocab(Vocab::encode("many"));
```

Comment:
I tried it on some modules, and it didn't seem to mess anything (at least on `autobiographicalMemory` ). But I'm not 100% sure.